<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>930</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>930</width>
       <height>630</height>
      </size>
     </property>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>741</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QWidget" name="pasajeros">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QTableWidget" name="tabla">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>70</y>
          <width>741</width>
          <height>371</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>RUT</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>NOMBRE</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>APELLIDO</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>CORREO</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>TELEFONO</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>HABITACION</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ENCARGADO</string>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>20</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>PASAJEROS</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_actualizar">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>460</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Actualizar</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="registrar">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>40</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Registro de pasajeros</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>100</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Rut:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>150</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>200</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Apellido:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>250</y>
          <width>55</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Correo:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>300</y>
          <width>60</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Telefono:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>370</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Habitacion:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_rut">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>100</y>
          <width>301</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_nombre">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>150</y>
          <width>301</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_apellido">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>200</y>
          <width>301</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_correo">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>250</y>
          <width>301</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lne_telefono">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>300</y>
          <width>301</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QCheckBox" name="encargado">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>370</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Encargado</string>
        </property>
       </widget>
       <widget class="QComboBox" name="habitaciones">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>370</y>
          <width>73</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_registrar">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>420</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Registrar</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="habitacion">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QLabel" name="img_const">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>30</y>
          <width>571</width>
          <height>431</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>img/img_const.png</pixmap>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="b2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>REGISTRAR PASAJEROS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>MOSTRAR PASAJEROS</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>HABITACIONES</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="img/img.qrc"/>
 </resources>
 <connections/>
</ui>
